docs/source/index.rst

```diff
@@ -24,21 +24,15 @@ Welcome to the Cozmo SDK Beta!
 
 .. toctree::
     :maxdepth: 2
-    :caption: Get Started
-
-    getstarted.rst
-
-.. toctree::
-    :maxdepth: 2
     :caption: Downloads
 
     downloads.rst
 
 .. toctree::
     :maxdepth: 2
-    :caption: Beginner Tutorial
+    :caption: Get Started
 
-    tutorial-beginner.rst
```

Comment:
With the latest changes in the branch, and after deleting docs/build, I don't see this warning anymore.